gen/generator.go

```diff
@@ -913,6 +1106,26 @@ func (m *Model) SetResourceNotesAndWarnigns(classPkgName string, definitions Def
 	}
 }
 
+func (m *Model) SetApicType(classDetails interface{}) {
+	platform := make(map[string][]interface{})
+	if platformFlavors, ok := classDetails.(map[string]interface{})["platformFlavors"].([]interface{}); ok {
+		platform["apicType"] = platformFlavors
+		m.PlatformFlavors = platform
+	}
+}
+
+func (m *Model) SetClassExclude() {
```

Comment:
do we use exclude anywhere?

Reply:
We don't use it explicitly in the classes.yml as of yet but within the generator itself.